kratos/includes/variables.h

```diff
@@ -412,7 +412,7 @@ namespace Kratos
 
     //for Structural application:
     KRATOS_DEFINE_VARIABLE( Matrix, INERTIA )
-
+    
```

Comment:
In the future avoid these kind of aesthetic changes in such a big PR. (probably is something related with the configuration of your code editor). These things make the review process much tedious.